<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serial</class>
 <widget class="QWidget" name="serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>314</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>234</width>
    <height>314</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>234</width>
    <height>314</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="com1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>fail</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pass">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>pass</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="com2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="com3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>com1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>com2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>com3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
